starter-code/index.html

```diff
@@ -9,6 +10,45 @@
   <script type="text/javascript" src="./js/index.js"></script>
   <title>Ironhack cart</title>
 </head>
+
 <body>
+  <h1>IronHack Merchandising Shop</h1>
+<div id="product-list">
+  <div class = "product">
+    <div class="name container"><span>IronBubble-head</span></div>
+    <div class="price container"><span>25.00</span></div>
+    <div class="qty container">
+            <label>QTY</label>
+            <input class="qty-input" value="0" min="0" max="100" type="number"></input>
```

Comment:
Nice work.  You even added an input for numbers to the users can click the little arrows to change the number